### PR TITLE
Rehearsal updates

### PR DESCRIPTION
**Pull request by jsafrane**
See individual commits, I think I fixed most of what I had noted from the rehearsal.

@davidz627, FYI, please merge it if it's fine. 

I did not fix these, because we're going to talk about them without slide:
- David: discovery of members of statefulset (-> they have stable names -> dns, maybe service?)
- Jan: recommend CSI over flex

Still remaining: ephemeral volumes (Pod with a secret or config map?)

### CODE REVIEW

File: storage101.md [hunk 1]
```diff
@@ -867,24 +789,24 @@ template: admin
...
 template: admin
 
-# PersistentVolume: Dynamic provisioning
+# PersistentVolume: Manual provisioning
 
-.center[
-  <img src="provisioning7.png" width="50%"/><br/>
-]
+- "Brownfield" use case.
+  - Using data of old apps.
```
Comment (davidz627) [inline storage101.md:820]: kind of a sparse slide? Maybe a quick image of what pre-provisioning "looks like"?

File: storage101.md [hunk 2]
```diff
@@ -867,24 +789,24 @@ template: admin
...
 
 ---
 
 template: admin
 
-# PersistentVolume: Dynamic provisioning
+# PersistentVolume Life Cycle: Release
```
Comment (davidz627) [inline storage101.md]: Are these the right titles?

File: storage101.md (hunk 1 above)
Comment (jsafrane) [reply storage101.md:820]: If I add a picture I shall describe it and I thought we're going to defer pre-provisioned PVs as too advanced.

I added the binding back, just in two pictures and I will try to be very quick when describing it

File: storage101.md (hunk 2 above)
Comment (jsafrane) [reply storage101.md]: good catch, I set all slides to `PersistentVolume: Dynamic Provisioning` (or Release etc)

**Comment by jsafrane** [comment]
I am merging this as it is, I need to generate PDF today and publish it on sched.org. We can still change the slides afterwards.

**Pull request merged**